.github/workflows/UPDATE_DEPENDENCIES.yml

```diff
@@ -25,13 +25,5 @@ jobs:
       - name: Create dependency updates PR
         uses: peter-evans/create-pull-request@v4
         with:
-          token: ${{ secrets.GH_TOKEN }}
-          commit-message: Update project dependencies
```

Comment:
The token defaults to the `GITHUB_TOKEN`, commit-message, title and body also have some defaults in place. The delete-branch key allows us to delete the branch if the PR is closed or merged by the bot and the branch-suffix just creates unique branches per PR.